src/nvim/edit.c

```diff
@@ -3419,6 +3419,7 @@ static void ins_compl_fixRedoBufForLeader(char_u *ptr_arg)
     else
       return;        /* nothing to do */
   }
+  assert(ptr != NULL);
```

Comment:
@justinmk If I am not mistaking, PVS studio thinks ptr can be NULL because it sees us checking for it not being NULL.